src/harness/util.py

```diff
@@ -125,3 +135,14 @@ def makePpaAndPalRecordsConsistent(ppa_record, pal_records, low_frequency,
   # Converting from defaultdict to dict
   ppa_record = json.loads(json.dumps(ppa_record))
   return ppa_record, pal_records
+
+
+def TriggerDailyActivitiesImmediatelyAndWaitUntilComplete(_sas_admin):
```

Comment:
Oh, was that the plan? :) I made my suggestions based on whether the function needed access to member variables/functions (asserts, _sas_admin) or not. Clearly we need to pick one and document it :)